src/game/camera.c

```diff
@@ -4551,15 +4660,21 @@ void play_camera_buzz_if_c_sideways(void) {
 }
 
 void play_sound_cbutton_up(void) {
+#ifndef REONUCAM
     play_sound(SOUND_MENU_CAMERA_ZOOM_IN, gGlobalSoundSource);
+#endif
 }
 
 void play_sound_cbutton_down(void) {
+#ifndef REONUCAM
     play_sound(SOUND_MENU_CAMERA_ZOOM_OUT, gGlobalSoundSource);
+#endif
 }
 
 void play_sound_cbutton_side(void) {
+#ifndef REONUCAM
     play_sound(SOUND_MENU_CAMERA_TURN, gGlobalSoundSource);
+#endif
```

Comment:
this would need further implementation but we could just define/undefine a camera turn sound? if not then maybe just `DISABLE_CAMERA_TURN_SOUND`
```suggestion
#ifndef CAMERA_TURN_SOUND
    play_sound(CAMERA_TURN_SOUND, gGlobalSoundSource);
#endif
```

Reply:
`#ifndef CAMERA_TURN_SOUND` should be `#ifdef`

Reply:
fixed